frontend/src/components/Navbar.js

```diff
@@ -19,7 +20,7 @@ const NavigationBar = (props) => {
   return (
     <>
       <div className="title-background">
-        <h1 onClick={() => setNavState("home")} className="title">saga</h1>
+        <img className="title" alt="saga logo" src={logo} onClick={() => setNavState("home")} />
```

Comment:
This might have some conflicts with me cuz I changed a little thing in Navbar.js and Navbar.css. Should I just copy-paste this info into my file commit and then pr?

Reply:
Yeah, that might be best. If it is only this one line, that should be fine.